Mathlib/Data/Finset/Sum.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yaël Dillies, Bhavik Mehta
 -/
 import Mathlib.Data.Finset.Card
-import Mathlib.Data.Finset.Union
```

Comment:
Why it `Union` a problematic import here? If we split `disjUnion` and `union` into separate files, would things become better?